APITaxi2/schemas.py

```diff
@@ -296,6 +301,11 @@ class TaxiSchema(Schema):
         required=False, allow_none=False,
         validate=validate.OneOf(UPDATABLE_VEHICLE_STATUS)
     )
+    # Adjustable visibility radius (if null, fallback to max radius)
+    radius = fields.Integer(
+        required=False, allow_none=False,
+        validate=validate.Range(min=TAXI_MIN_RADIUS, max=TAXI_MAX_RADIUS)
```

Comment:
Don't you think it is a good idea to allow null, to know which users decide to set the radius to the maximum, and which don't set the radius at all?